packages/modules/create-imodel/src/components/upload-image/UploadImage.tsx

```diff
@@ -44,7 +45,15 @@ export type UploadImageProps = {
   };
 };
 
-export function UploadImage({
+export function UploadImage(props: UploadImageProps) {
+  return (
+    <ThemeProvider>
```

Comment:
wow..
this is just bad with extra divs randomly..
@mayank99 Can this be fixed?

Reply:
can what be fixed? you don't need `ThemeProvider` in every single component, you only need it once per tree.

if all components are separate/standalone entrypoints, then this might just be the best way.

Reply:
@veekeys I see that this component now uses the `useIModelComponent` hook which makes this component essentially only work within the Create/Update components, is that the case ? I assumed it was meant to be used by itself but if it is not the case we should be ok removing this Wrap here.

Reply:
yeah, I think we should not allow using `UploadImage` component outside of create/update components. Probably can get rid of it